docs/.vuepress/config/markdown.js

```diff
@@ -2,5 +2,10 @@ const markdown = {
   extendMarkdown: md => {
     // use more markdown-it plugins!
     md.use(require('markdown-it-include'));
+    md.use(require('markdown-it-multimd-table'), {
```

Comment:
Maybe we won't need this. It's mostly if we need to document complex tables (see [this commit](https://github.com/strapi/documentation/pull/1035/commits/86c4bafa45d1cabaa9a697e0913cdc9b8e4e189c) which adds support for MultiMarkdown tables.